app/web/src/components/Hero/Hero.tsx

```diff
@@ -13,153 +13,217 @@ import ProjectsCell from 'src/components/ProjectsCell'
 import OutBound from 'src/components/OutBound/OutBound'
 
 // dynamic import to enable pre-render iof the homepage
-const Coffee = React.lazy(() => import('src/components/Hero/AssetWithGooey'))
+const AssetWithGooey = React.lazy(() => import('src/components/Hero/AssetWithGooey'))
+const cqCode = `from cadquery import *
 
-export const Hero = () => {
-  const [width, widthSetter] = React.useState(1024)
+d1 = 58.5
+d2 = 56.5
+d3 = 63.5
+d4 = 84
+d5 = 88
 
-  React.useEffect(() => {
-    const onResize = () => {
-      widthSetter(window.innerWidth)
-    }
-    window.addEventListener('resize', onResize)
-    onResize()
-    return () => {
-      window.removeEventListener('resize', onResize)
-    }
-  }, [])
-  const { heroOffset, tutOffset } = React.useMemo(() => {
-    if (width < 1024) {
-      return {
-        heroOffset: [0, -3],
-        tutOffset: [0, -3],
```

Comment:
this was used to move the asset in 3d when screen size changes and can be removed now.